src/user/user.service.ts

```
@@ -0,0 +1,31 @@
+import { Inject, Injectable } from '@nestjs/common';
+import { CreateUserDto } from './dto/create-user.dto';
+import { UpdateUserDto } from './dto/update-user.dto';
+import { DRIZZLE } from 'src/drizzle/drizzle.module';
+import type { DrizzleDB } from 'src/drizzle/types/drizzle';
+import { users } from 'src/drizzle/schema/users.schema';
```

ESLint

[error] 'users' is defined but never used.

(@typescript-eslint/no-unused-vars)